hack/local-up-karmada.sh

```diff
@@ -75,8 +75,8 @@ fi
 util::cmd_must_exist "go"
 util::verify_go_version
 
-# Make sure docker exists
-util::cmd_must_exist "docker"
+# Make sure docker is available
+util::verify_docker
```

Comment:
Nice finding! Just a small question, does the `usage` function https://github.com/karmada-io/karmada/blob/65376b28d5037c27ff7ec0e56542c2a345d1a120/hack/local-up-karmada.sh#L26 need to add a description of prerequisites for docker is available?

Reply:
During my trip, I set up the environment according to the prerequisites in the README. refer:  https://github.com/karmada-io/karmada/blob/65376b28d5037c27ff7ec0e56542c2a345d1a120/README.md?plain=1#L100
So would it be better to put it in the README? I think the `usage` function should describe functions of scripts or usage options of scripts.

And so far there is no description of prerequisites in usage. If we add docker-related descriptions to usage, do we also need to add related descriptions to other prerequisites?

Reply:
> So would it be better to put it in the README? I think the `usage` function should describe functions of scripts or usage options of scripts.

That makes sense to me

Reply:
> So would it be better to put it in the README? I think the `usage` function should describe functions of scripts or usage options of scripts.

Are you interested in submitting a PR for this?

Reply:
> > So would it be better to put it in the README? I think the `usage` function should describe functions of scripts or usage options of scripts.
> 
> Are you interested in submitting a PR for this?

Sorry for the late reply.
I think it's a bit strange to add a "make sure docker is available" in prerequisites.
It feels incompatible with existing items. https://github.com/karmada-io/karmada/blob/master/README.md?plain=1#L100,L103
What do you think?

Reply:
>  What do you think?

How about 
```
- [docker](https://www.docker.com/) version vx.x.x+ 
note: execute command `docker ps` to make sure docker is available
```

Reply:
> > What do you think?
> 
> How about
> 
> ```
> - [docker](https://www.docker.com/) version vx.x.x+ 
> note: execute command `docker ps` to make sure docker is available
> ```

This is OK.
But since I'm new to the project, I'm currently not sure about the docker version. Can we determine the minimum docker version? Alternatively, is there a way to help me to determine the lowest docker version?

Reply:
I don't think it's a good idea to list a long list of prerequisites in README.
It's common sense that have docker installed.

Reply:
ok, Let's do less first.
If it's really confusing here, we can add more details.
OK?

Reply:
Sure. Thanks @zll600. 
Now everything looks fine. Let's wait and see if others need more details.